### PR TITLE
Add participant Krishna Singh

### PR DESCRIPTION
**Pull request by ultimatekrishna**
I would like to register for JS CraftCamp.

- [x] My pull request contains a JSON file `$name_or_nickname.json`
- [x] I read the `THE IMPORTANT STUFF` at [https://jscraftcamp.org/registration](https://jscraftcamp.org/registration), the JSON file follows the [template](https://github.com/jscraftcamp/website/blob/main/participants/_template.json), and contains the mandatory fields like `realName`, `githubAccountName`, `when`, `iCanTakeNotesDuringSessions`, `whatIsMyConnectionToJavascript` and `whatCanIContribute`.
- [x] If I can NOT attend, I will either send another pull request removing my JSON file or an e-mail with the subject 'UNREGISTER' to team@jscraftcamp.org
- [x] I agree that the data I enter in the registration can be used for running the event, e.g. make a name tag, count me as participant. I acknowledge that my data are public on GitHub and I agree that I will be listed on the JSCraftCamp website as a participant.
- [x] I agree that photos and videos might be taken and published (e.g. on social media) during the event.
- [x] I understand that I might NOT get a T-Shirt, because they might be in production already
- [ ] I asked my company about sponsoring (see open items at: https://github.com/orgs/jscraftcamp/projects/5/views/1)

Are you from a sponsoring company?

- [ ] Yes, I am from company ?????

### CODE REVIEW

File: participants/krishnasingh.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	// your current interests (JS and in general) (required)
+	"tags": ["Javascript", "Angular", "React", "Micro-Frontends", "Webpack", "Node"],
+	// if you only eat vegan food (optional)
+	"vegan": false,
+	// if you only eat vegan or vegetarian food (optional)
+	"vegetarian": false,
+	// what you cannot eat or drink (optional); If you don't want to put it in here, message the organizers.
+	// IMPORTANT: we cannot guarantee that food for every diet will be available,
+	//            if you have gluten free diet, please make backup plans.
+	"allergies": ["-"],
```
Comment (meldron) [inline participants/krishnasingh.json:36]: Just leave the array empty

File: participants/krishnasingh.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	// the following sizes are available: S, M, L, XL, 2XL, 3XL (only regular cut)
+	"tShirtSize": "S",
+	// your Mastodon URL (optional)
+	"mastodon": "",
+	// your LinkedIn profile URL
+	"linkedin": "https://www.linkedin.com/in/ultimatekrishna/",
+	// your X (Twitter) handle - must match regex ^[a-zA-Z_]{1}[a-zA-Z0-9_]{0,14}$ (optional)
+	"X": "-",
+	// your website URL or other social media (optional)
+	"website": "-"
```
Comment (meldron) [inline participants/krishnasingh.json:51]: Leave the string empty or just remove the properties all together.

**Review by meldron — CHANGES_REQUESTED** [review]
Hi @ultimatekrishna,

looking forward to see you at JSCC. You just have to make some tiny adjustments to your registration json. If you don't have an account or allergy, just leave the string empty, set it to `null` or remove the property.

File: participants/krishnasingh.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+	// your current interests (JS and in general) (required)
+	"tags": ["Javascript", "Angular", "React", "Micro-Frontends", "Webpack", "Node"],
+	// if you only eat vegan food (optional)
+	"vegan": false,
+	// if you only eat vegan or vegetarian food (optional)
+	"vegetarian": false,
+	// what you cannot eat or drink (optional); If you don't want to put it in here, message the organizers.
+	// IMPORTANT: we cannot guarantee that food for every diet will be available,
+	//            if you have gluten free diet, please make backup plans.
+	"allergies": [""],
```
Comment (Narigo) [inline participants/krishnasingh.json:36]: Hi @ultimatekrishna , the test was still failing, see https://github.com/jscraftcamp/website/actions/runs/9007558022/job/24756843703

If there are no allergies, please use an empty array or remove the whole field.

```suggestion
	"allergies": [],
```

**Review by lowsky — APPROVED** [review]
Looks good, CU at jscraftcamp

**Review by lowsky — APPROVED** [review]

**Pull request merged**